Developer Samples/LibraryLogEnabler/LibLogEnabler/App_Packages/LibLog.4.2/LibLog.cs

```diff
@@ -40,23 +40,23 @@
 
 using System.Diagnostics.CodeAnalysis;
 
-[assembly: SuppressMessage("Microsoft.Design", "CA1020:AvoidNamespacesWithFewTypes", Scope = "namespace", Target = "TraceLogger.Logging")]
-[assembly: SuppressMessage("Microsoft.Design", "CA1026:DefaultParametersShouldNotBeUsed", Scope = "member", Target = "TraceLogger.Logging.Logger.#Invoke(TraceLogger.Logging.LogLevel,System.Func`1<System.String>,System.Exception,System.Object[])")]
+[assembly: SuppressMessage("Microsoft.Design", "CA1020:AvoidNamespacesWithFewTypes", Scope = "namespace", Target = "LibLogEnabler.Logging")]
```

Comment:
Were there customizations made to this file, or was this change applied by the nuget package's install tooling?

Reply:
nuget upgrade automatically updated everything from TraceLogger to LibLogEnabler